otter/effect_dispatcher.py

```diff
@@ -28,3 +32,15 @@ def get_dispatcher(reactor):
         }),
         make_twisted_dispatcher(reactor),
     ])
+
+
+def get_full_dispatcher(reactor, authenticator, log, service_mapping, region):
```

Comment:
I'm... not sure what you mean. Are you asking why there's both a get_simple_dispatcher and a get_full_dispatcher? get_simple_dispatcher is pretty much only used for tests, but also temporarily in places like _rcv3.py. It should either be deleted or be relegated to purely a testing utility soon.